cicd/gitops/run-gitops-control-plane-workload-on-infra-nodes.adoc

```diff
@@ -0,0 +1,13 @@
+[id="run-gitops-control-plane-workload-on-infra-nodes"]
+= Running GitOps control plane workload on infra nodes
```

Comment:
```suggestion
= Running GitOps control plane workloads on infra nodes
```
Or `Running a GitOps control plane workload on infra nodes` if you mean to leave it as singular rather than plural